src/h5web/visualizations/line/models.ts

```diff
@@ -1,6 +1,7 @@
 import circleURL from './sprites/circle.png';
 import crossURL from './sprites/cross.png';
 import squareURL from './sprites/square.png';
+import capURL from './sprites/cap.png';
```

Comment:
Is it on purpose that the image is square with transparent zones? Could it just be rectangular without transparency? Might save a few bytes.

Reply:
Yep. Putting a simple rectangular image leads to this:
![image](https://user-images.githubusercontent.com/42204205/101601403-ef0bc580-39fc-11eb-8330-9866cb08f499.png)

Reply:
Oh right, I get it 👍